src/app/pages/storage2/modules/devices/components/disk-info-card/disk-info-card.component.html

```diff
@@ -0,0 +1,75 @@
+<mat-card class="container-card">
+  <mat-card-header>
+    <h3 mat-card-title>{{ 'Disk Info' | translate }}</h3>
+    <button
+      mat-button
+      *ngIf="!!diskInfo"
+      (click)="onEdit()"
+    >{{ 'Edit' | translate }}</button>
+  </mat-card-header>
+  <mat-card-content>
+    <ng-container *ngIf="loading || !!diskInfo; else noDiskInfo">
+      <div class="row">
+        <span class="header">{{ 'Disk Size' | translate }}:</span>
+        <span *ngIf="!loading; else loaderSkel">
+          {{ diskInfo?.size ? formatter.convertBytesToHumanReadable(diskInfo.size) : 'N/A' }}
+        </span>
+      </div>
+      <div class="row">
+        <span class="header">{{ 'Transfer Mode' | translate }}:</span>
+        <span *ngIf="!loading; else loaderSkel">
+          {{ diskInfo?.transfermode || 'N/A' }}
+        </span>
+      </div>
+      <div class="row">
+        <span class="header">{{ 'Serial' | translate }}:</span>
+        <span *ngIf="!loading; else loaderSkel">
+          {{ diskInfo?.serial || 'N/A' }}
+        </span>
+      </div>
+      <div class="row">
+        <span class="header">{{ 'Model' | translate }}:</span>
+        <span *ngIf="!loading; else loaderSkel">
+          {{ diskInfo?.model || 'N/A' }}
+        </span>
+      </div>
+      <div class="row">
+        <span class="header">{{ 'Rotation Rate' | translate }}:</span>
+        <span *ngIf="!loading; else loaderSkel">
+          {{ diskInfo?.rotationrate ? diskInfo.rotationrate + ' ' + ('RPM' | translate) : 'N/A' }}
```

Comment:
can be like `'{n} RPM' | translate: { n: diskInfo.rotationrate }` not sure if needed